docs/docs/exosphere/register-node.md

```
@@ -28,6 +30,32 @@ class MyNode(BaseNode):
         pass
 ```
 
+
+### Node Lifecycle Architecture
+
+```mermaid
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)